settings.md

```diff
@@ -17,21 +17,38 @@ title: Kint settings
 
 # Kint settings
 
-These are global Kint settings. For plugin settings please see the [plugins page]({{ site.baseurl }}/plugins/).
+These are global Kint settings. For plugin settings please see the [plugins page]({{ site.baseurl }}/plugins/). 
 
 <section id="common" markdown="1">
 
+## Usage with a framework
+
+Chances are you're using `composer`, in that case the cleanest way is to add an entry to the `autoload.files` configuration:
+
+```json
+    "autoload": {
+        "psr-4": {
+            "App\\": "src/"
+        },
+        "files": [
+            "config/kint.php"
+        ]
+    },
+```
+
+Place your settings there (`"config/kint.php"`). [Here is what settings I personally use](https://gist.github.com/raveren/eba373d8abb572b0528c73d145103f95).
```

Comment:
I'm making a 80% improvement with 20% of the effort here, I don't know what does blacklisting the DI containers would do, either.

Reply:
DI containers tend to contain _every_ instance in your application (Think symfony/container) so if kint comes across it it'll end up trying to dump a _lot_ of stuff. Is a bit of a perf killer in modern applications, so you want to blacklist it, but even though it's used a lot it's not core PHP so I didn't add it to the docs or as a default. It would fit in a separate page though. (So would a `dd` reimplementation)

If there's something wrong with the default config we should fix it in the default config, if not it feels like "Extra"

In your gist you've got:

* Disable folder: Default in next
* PhpStorm: Definitely not core PHP. Sublime text uses `subl://open?url=file://%f&line=%l` for instance, and I don't want to add editor-specific settings.
* Helpers: I have nothing against adding helpers, but I want to keep them to a minimum in vanilla, and I don't want to write up extra docs for them either

I still think they'd be better under separate headings on a separate page, that way if we're going to offer bits of your-mileage-may-vary code then people can pick and choose which parts they want